lgsf/aws_lambda/handlers.py

```diff
@@ -13,25 +15,76 @@ def scraper_worker_handler(event, context):
     council = message["council"]
     command_name = message["scraper_type"]
     scraper_cls = load_scraper(council, command_name)
-    options = {"council": council, "verbose": True}
+    options = {"council": council, "verbose": True, "aws_lambda": True}
     console = Console(file=sys.stdout)
     scraper = scraper_cls(options, console)
-    scraper.run()
+    # scraper.run()
+
+    codecommit_client = boto3.client("codecommit")
+
+    main_info = codecommit_client.get_branch(
+        repositoryName="CouncillorsRepo", branchName="main"
+    )
+
+    now = datetime.now().strftime("%Y-%m-%d")
+    branch_name = f"{council}-{now}"
+
+    codecommit_client.create_branch(
+        repositoryName="CouncillorsRepo",
+        branchName=branch_name,
+        commitId=main_info["branch"]["commitId"],
```

Comment:
Not very important, but do we care about race conditions here? What if `main` has a newer commit between line 25 and this line?

Reply:
Good question - I was hoping that becasue the scrapers will have been commiting to different files it won't matter, and will still be fast forward-able. However I haven't tried this out.